Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeTokenizationClient.swift

```diff
@@ -16,7 +16,7 @@ class BTPayPalNativeTokenizationClient {
 
         let tokenizationRequest = BTPayPalNativeTokenizationRequest(returnURL: returnURL,
                                                                     request: request,
-                                                                    correlationID: "", // TODO: get this value from PayPalCheckout SDK
+                                                                    correlationID: State.correlationIDs.riskCorrelationID ?? "",
```

Comment:
Question: Is defaulting to an empty string appropriate here or should we return an error to the merchant if we don't have a correlation ID at this point?

Reply:
Didn't we talk about this in stand? We decided to go with erroring if there is no `riskCorrelationID` from the Native XO right? @sshropshire can you confirm?

Reply:
I added a TODO for this and am going to add a JIRA for this work. `State.correlationIDs.riskCorrelationID` is proving to be pretty tricky to mock.